### DISCUSSION TITLE
Questions about mainland China

### DISCUSSION THREAD

**Discussion by kevinluo6191** [community/community#77537] [GitHub Education]
### Hi everyone,

Hi everyone,I am located in mainland China and am a student at the University of International Business and Economics (UIBE). Due to the situation in mainland China, I need to use a VPN to log in to github. Is there a way for me to apply for the github student plan?

**Comment by hackelite01** [comment]
Hey @kevinluo6191 , I can do it easily. Drop the details on mail at [hackelite.sup@gmail.com](mailto:hackelite.sup@gmail.com)
